src/components/iap/stripe-legacy-iap-prices.json

```diff
@@ -0,0 +1,14 @@
+{
+  "collectionName": "components_iap_stripe_legacy_iap_prices",
+  "info": {
+    "displayName": "stripeLegacyIapPrices",
```

Comment:
To be consistent with the other components' display names - `Stripe Legacy IAP Prices`